core/domain/exp_jobs_one_off.py

```diff
@@ -160,7 +161,7 @@ def map(item):
                             # special characters like sqrt and pi, which is why
                             # they need to be replaced with their corresponding
                             # text values before performing validation.
-                            for unicode_char, text in UNICODE_TO_TEXT.items():
+                            for unicode_char, text in MathExpressionValidationOneOffJob.UNICODE_TO_TEXT.items(): #pylint: disable=line-too-long
```

Comment:
Avoid pylint pragmas. You can define a local var with a shorter name, for example.

In general please try to really avoid pylint pragmas in your code, going forward. The first thing the reviewer will do is to ask you why you need them in the first place.

Reply:
Understood. I've changed it now.